### DISCUSSION TITLE
Is there a way to bulk copy column labels between different views?

### DISCUSSION THREAD

**Discussion by danielrosehill** [Budibase/budibase#13984] [Help]
Let's say that I modify a bunch of columns like this by creating readable display labels:

![column display values](https://github.com/Budibase/budibase/assets/40233203/211f3977-a821-44d0-8377-68ff2c6c42d6)

If I want to create a new page for modifying the data ... is there a way to do this without having to start the process of labelling them from scratch? Can the modified labels be stored and quickly replicated between pages and views?

**Comment by ConorWebb96** [comment]
Hey @danielrosehill,

What happens if you copy the component from the component tree and then paste it into a new screen. You can't copy components between apps, but I'm pretty sure you can copy components between screens in the same app.

Hopefully this works for you.
